### ISSUE TITLE
Line number reporting doesn't work with versions of Word where UIA is enforced

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#13537]
### Steps to reproduce:
1. Enable line number reporting (via NVDA settings, document formatting)
2. Start Word
3. Type some lines

### Actual behavior:
Arrow down
Note Line numbers are not reported.
E.G.
```
This is a test
down arrow
With multiple lines
```

### Expected behavior:
Line numbers are reported, EG

```
line 1  This is a test
down arrow
line 2  With multiple lines
```

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-25076,14a94998

#### Windows version:
Windows 11 21H2 (OS Build 22000.556)

#### Name and version of other software in use when reproducing the issue:
Word 365 16.0.15028.20094

**Comment by XLTechie** [comment]
Possible duplicate of #11248.

I was just discussing this via email with @Qchristensen yesterday.

As a point of curiosity:

I know it doesn't work, but if it did, would you be expecting NVDA's idea of the 
line numbers here, or Word's (I.E. the numbers shown if you turn on line 
numbering under page layout)?

Currently, not using UIA, NVDA seems to be indicating the line number as it 
appears in the status bar, not the left edge line numbers as indicated by 
continuous line numbering set under layout.
I consider that a bug, but apparently there is some debate about this.

**Comment by michaelDCurran** [comment]
@leonardder  I know that Microsoft did role back this feature for several days when the crash was identified   in MS Word when NVDA 2022.1beta1 came out. Once we released beta2, they re-enabled this feature across installs of Office. Please ensure your MS Word is fully up to date. (No matter your version number), MS can apparently disable and enable these features such as custom patterns.

**Comment by leonardder** [comment]
Thanks for the clarification. Just started Word and out of the blue, it works again.
